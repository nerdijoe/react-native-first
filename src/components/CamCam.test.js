import React from 'react'

import renderer from 'react-test-renderer'


import mockCamera from '../__mocks__/Camera';
jest.mock('react-native-camera', () => mockCamera);

import CamCam from './CamCam'


describe('CamCam', () => {
  it('render without crashing', () => {
    const tree = renderer.create(<CamCam />)
    expect(tree).toMatchSnapshot()
  });

});
