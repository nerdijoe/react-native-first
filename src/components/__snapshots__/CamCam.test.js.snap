// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CamCam render without crashing 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }
/>
`;
